system76-firmware/riscv64.patch

```diff
@@ -0,0 +1,34 @@
+diff --git PKGBUILD PKGBUILD
+index 5b7a1f5..bbd016b 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -8,7 +8,7 @@ pkgrel=1
+ pkgdesc='System76 Firmware Tool and Daemon'
+ arch=(x86_64)
+ url=https://github.com/pop-os/system76-firmware
+-license=(GPL3)
++license=(GPL-3.0-only)
+ depends=(
+   ca-certificates
+   dbus
+@@ -26,8 +26,10 @@ makedepends=(
+ provides=(system76-firmware-daemon)
+ conflicts=(system76-firmware-daemon)
+ _tag=6e98acaa75c05344440d10f8fb924da0e2406238
+-source=(git+https://github.com/pop-os/system76-firmware.git#tag=${_tag})
+-b2sums=('92b90259e67c3e339ab9a93b5f850e5b0fdadbe72971c3b904218b3c60c978382adbc278c02eaf325a4c187cf4503c32b6bd909e8b099dd213774bc16b3e3923')
++source=(git+https://github.com/pop-os/system76-firmware.git#tag=${_tag}
++        fix-riscv64-io.patch)
++b2sums=('92b90259e67c3e339ab9a93b5f850e5b0fdadbe72971c3b904218b3c60c978382adbc278c02eaf325a4c187cf4503c32b6bd909e8b099dd213774bc16b3e3923'
++        '61982e61d6cebf558d032046f3bbf162d027d792f6d367c8371f7dfbdff6adcafdea70610bbdd85c4506aa2f06ad11f1103662e9e7778adc0db88ad08c148b18')
+ 
+ pkgver() {
+   cd system76-firmware
+@@ -36,6 +38,7 @@ pkgver() {
+ 
+ build() {
+   cd system76-firmware
++  patch -Np1 -i ../fix-riscv64-io.patch
```

Comment:
Patching should be done in a separate `prepare()` function instead of in `build()`.